gst/nnstreamer/tensor_typedef.h

```diff
@@ -133,6 +133,17 @@ typedef union {
   uint64_t _uint64_t;
 } tensor_element;
 
+/**
+ * @brief Internal tensor layout format for other/tensor
+ */
+typedef enum _nns_tensor_layout
+{
+  _NHWC = 0,    /**< NHWC: channel last layout */
+  _NCHW,        /**< NCHW: channel first layout */
+
+  _none_layout,        /**< Layout unknown or not specified */
+} tensor_layout;
```

Comment:
I think it's better for this enum to also have a similar format with 
```
typedef enum _ml_tensor_layout_e
{
  ML_TENSOR_LAYOUT_NHWC = 0,      /**< Channel first layout */
  ML_TENSOR_LAYOUT_NCHW,          /**< Channel last layout */
  ML_TENSOR_LAYOUT_NONE,          /**< Unknown layout */
}
```
For example, like `NNS_TENSOR_LAYOUT_NHWC/NCHW/NONE`?

Reply:
yes, that's better - updated to `_NNS_LAYOUT_*`